src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -214,6 +217,10 @@ public static function fromDsn(string $dsn, array $options = [], AmqpFactory $am
             return $queueOptions;
         }, $queuesOptions);
 
+        if ($useAmqps && !self::hasCaCertConfigured($amqpOptions)) {
+            throw new SSLConnectionException("No ca certificate has been provided, set it in php files configuration 'amqp.cacert' or through DSN with cacert parameter to use SSL.");
```

Comment:
I would read better as such:

```
No CA certificate has been provided. Set "amqp.cacert" in the PHP configuration files or pass the "cacert" parameter in the DSN to use SSL.
```

Note that I'm using quotes rather than apostrophs in the message to quote the parameter names. this has 2 benefits:
- it is semantically correct (`'` is called a quote only by devs, not by human languages AFAIK)
- it matches the convention used in the WebProfilerBundle UI to emphasis such parts of error messages

Reply:
Ok, I've been using backslash to escape quotes, is it the correct one usage ?